client/ayon_core/lib/path_tools.py

```diff
@@ -210,18 +185,18 @@ def get_last_version_from_path(path_dir, filter):
     assert isinstance(filter, list) and (
         len(filter) != 0), "`filter` argument needs to be list and not empty"
 
-    filtred_files = list()
+    filtered_files = list()
 
     # form regex for filtering
     pattern = r".*".join(filter)
 
     for file in os.listdir(path_dir):
         if not re.findall(pattern, file):
             continue
-        filtred_files.append(file)
+        filtered_files.append(file)
 
-    if filtred_files:
-        sorted(filtred_files)
-        return filtred_files[-1]
+    if filtered_files:
+        sorted(filtered_files)
```

Comment:
BTW This line does nothing now.
```suggestion
        filtered_files.sort()
```